TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -97,6 +98,11 @@ private extension HomeViewController {
     @IBAction func uploadsButtonPressed() {
         viewModel?.uploadListSelected()
     }
+
+    @IBAction func viewTreesButtonPressed() {
+        viewModel?.viewTreesSelected()
+        print("view trees button pressed")
```

Comment:
Can we remove this print statement - feel free to use the Logger in the main branch though if you want to log something to the console in debug only.

Reply:
removed